cms/summary_probe

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# Copyright 2012-2020 CERN
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Fernando Garzon, <oscar.fernando.garzon.miguez@cern.ch>, 2021
+
+'''
+Probe to check used space.
+'''
+
+import sys
+import traceback
+
+from prometheus_client import CollectorRegistry, Gauge, push_to_gateway
+from rucio.common.config import config_get
+from rucio.core.rse import list_rses, get_rse_usage
+from rucio.db.sqla import models
+from rucio.db.sqla.session import get_session
+
+# Exit statuses
+OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
+
+PROM_SERVERS = config_get('monitor', 'prometheus_servers', raise_exception=False, default='')
+if PROM_SERVERS != '':
+    PROM_SERVERS = PROM_SERVERS.split(',')
+
+if __name__ == '__main__':
+    try:
+        registry = CollectorRegistry()
+        session = get_session()
+        g1 = Gauge('judge_used_space_rucio_per_RSE_per_Account'
```

Comment:
Two comments. The commas should end the previous line, not start a new one. And can you name your gauges something in English (dynamic_gauge)?